packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js

```diff
@@ -0,0 +1,4854 @@
+    ),
+    repo: "React",
+    fixedWithCodeMod: true
+  }, {
+    component: "Wizard",
+    description: (<>Deprecated the <code  className='ws-code'>Wizard</code> components and replaced them with a newer implementation.</>),
+    pullRequestURL: "https://github.com/patternfly/patternfly-react/pull/8821",
+    details: (
+      <TextContent>
+        <Text component={TextVariants.p}>Codemods will update import and export paths to our deprecated directory, but we recommend replacing it with our new <code className='ws-code'>Wizard</code> implementation.</Text>
+        <Text component={TextVariants.h2}>Example in</Text>
+        <CodeBlock>
+          <CodeBlockCode>
+            {`import { Wizard, Button } from '@patternfly/react-core';`}
+          </CodeBlockCode>
+        </CodeBlock>
+        <Text component={TextVariants.h2}>Example out</Text>
+        <CodeBlock>
+          <CodeBlockCode>
+            {`import { Button } from '@patternfly/react-core';
+import { Wizard } from '@patternfly/react-core/deprecated';`}
+          </CodeBlockCode>
+        </CodeBlock>
+      </TextContent>
+    ),
+    repo: "React",
+    fixedWithCodeMod: true
+  }, {
+    component: "Wizard",
+    description: (<>Updated the order of the "next" and "back" buttons so that the "next" button is rendered after the "back" button. This change affects the deprecated implementation and the <code  className='ws-code'>WizardFooter</code> component in the new implementation.</>),
+    pullRequestURL: "https://github.com/patternfly/patternfly-react/pull/8409",
+    repo: "React",
+    fixedWithCodeMod: false
+  }, {
+    component: "Label/Label group",
+    description: "Used gap instead of item margins",
+    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5221",
+    details: "Updated the label component to use CSS `gap` to space its children instead of using `margin` (same as chip).",
+    repo: "HTML/CSS",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Label/Label group",
+    description: " Add wrappers for content and actions to manage styling embedded components",
+    pullRequestURL: " https://github.com/patternfly/patternfly/pull/5360",
+    details: "Added wrapping elements for both the label content and actions to make the layout more flexible. (same as chip).",
+    repo: "HTML/CSS",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Label/Label group",
+    description: " Allow text to truncate",
+    pullRequestURL: " https://github.com/patternfly/patternfly/pull/5364",
+    details: "Updated the HTML structure so that text truncation is supported by default.",
+    repo: "HTML/CSS",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Layouts",
+    description: "Updated level, split, stack to use gap",
+    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5585",
+    details: "Updated the “gutter” variation of these layouts to use CSS `gap` to create gutters between children instead of `margin`",
```

Comment:
```suggestion
    details: "Updated the “gutter” variation of these layouts to use CSS `gap` to create gutters between children instead of `margin`.",
```